### ISSUE TITLE
rotate training images 

### ISSUE THREAD

**Issue by andrewljohnson** [trailbehind/DeepOSM#24]
## Benefits
- produce more training data
- make analysis more applicable using one data set to train, and another to set (less overfit locally and regionally/globally hopefully)
## Notes
- each rotation needs to be unique, so that the same image/label pair doesn't accidentally get put into both training and test data
